### ISSUE TITLE
ffmpeg (+depending packages) broken on octeon (buildbot)

### ISSUE THREAD

**Issue by plntyk** [openwrt/packages#852]
octeon is mips64
However the in the Makefile
```
ifeq ($(CONFIG_CPU_TYPE),"mips32")
FFMPEG_CONFIGURE +=--disable-mips32r2
endif
```
does not cover this case - leaving mips32r2 compile flag enabled

```
libavformat/aacdec.c:1:0: error: '-mips32r2' conflicts with the other architecture options, which specify a mips64r2 processor
```
http://buildbot.openwrt.org:8010/broken_packages/octeon/ffmpeg/compile.txt

**Comment by thess** [comment]
I get a fix for this tomorrow

**Comment by thess** [comment]
Should be fixed now by: e74ed31e760b50900412cef2e41ceacd3c469bb6

**Comment by plntyk** [comment]
verified as fixed:
-local compile: no error
-buildbot: no error
